### DISCUSSION TITLE
侧端测试单通道灰度图片效果很差

### DISCUSSION THREAD

**Discussion by dlkht** [PaddlePaddle/PaddleOCR#12182] [Q&A] [answered]
用端侧部署测试；
输入单通道文字图片；
预测结果很差，但三通道彩色图片的预测效果很好。
后来改了下程序中的mean/scale参数，结果有些变动。

请问paddleocr 用于单通道灰度图片文字识别时，应该怎么修改相关参数？

**Comment by tink2123** [comment, marked as answer]
由于训练时的图片都是三通道的，所以对单通道图片预测精度可能较低。模型参数已经固化了，没有办法通过修改参数来调整，可以在预测的时候把图片通过前处理变成3通道的。

**Comment by dlkht** [comment]
好的，谢谢！
